paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenInteractor.kt

```diff
@@ -0,0 +1,28 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCode
+import com.stripe.android.paymentsheet.viewmodels.BaseSheetViewModel
+import com.stripe.android.ui.core.cbc.CardBrandChoiceEligibility
+import com.stripe.android.uicore.utils.mapAsStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+internal interface ManageScreenInteractor {
+    val state: StateFlow<State>
+
+    data class State(
+        val paymentMethods: List<PaymentMethod>?,
+        val nameProvider: (PaymentMethodCode?) -> String,
```

Comment:
Can we resolve these before we send them off? Ideally this "logic" moves to the interactor, or somewhere out of the UI.  -- Just an idea/question, non blocking.